.travis.yml

```diff
@@ -143,7 +154,7 @@ script:
   - |
     case "${TRAVIS_OS_NAME}:${CFG}" in
       linux:|osx:|windows:)
-        ${FBUILD_CMD} Tests
+        no_timeout ${FBUILD_CMD} Tests
```

Comment:
I don't know too much about Travis, but I saw it has a built-in "travis_wait" command: https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received

Can that be used instead?